"docs(agw): fixing instructions to connect enodeb to gateway VM" (#13016)
magma/magma: docs/readmes/lte/dev_notes.md

To connect a physical eNodeB to the gateway VM:

1. Connect the eNodeB to a port on the host machine, say it is interface `en9`.

> 1. From the VirtualBox GUI, switch the Adapter 2 (for `eth2` interface) from

Reviewer: The interface name within the dev VM for S1 connection is `eth1`. VirtualBox GUI marks it as Adapter 2
```suggestion
1. From the VirtualBox GUI, switch the Adapter 2 (for `eth1` interface) from
```

`Host-only` to `Bridged` mode and bridge it to interface `en9` from above.
1. In gateway VM, modify the `nat_iface` in `/etc/magma/pipelined.yml` from
`eth2` to `eth0`. Restart all services.
1. In the gateway VM, follow the steps in [EnodeB
Configuration](enodebd#basic-troubleshooting). Make sure the `earfcn` set in the
enodebd section of `gateway.mconfig` is the one that is supported by the eNodeB
under consideration. Use this [calculator](https://www.sqimway.com/lte_band.php) to

> get the `earfcn` corresponding to the frequency range specified by the eNodeB.

Reviewer: ```suggestion
get the `earfcn` corresponding to the frequency range listed on the eNodeB.
```
